internal/errors/redis.go

```diff
@@ -73,10 +73,7 @@ type ErrRedisNotFoundIdentity struct {
 
 // Error returns the string of ErrRedisNotFoundIdentity.error.
 func (e *ErrRedisNotFoundIdentity) Error() string {
-	if e.err != nil {
-		return e.err.Error()
-	}
-	return ""
+	return e.err.Error()
```

Comment:
What about the previous author's feedback as to whether the error should be exported? :thinking:

Reply:
I think it should return an error string as same as golang's default  `func(e *error) Error() string{}`.

Reply:
I'm sorry, my way of communicating was not good. 
I wanted to tell you whether it should be the following A implementation(exported error) or the following B(Unexported error) implementation.

A. (Exported error)
```go
type ErrRedisNotFoundIdentity struct {
  Err error <- this
}

func (e *ErrRedisNotFoundIdentity) Error() string {
...
}
```


B. (Unexported error)
```go
type ErrRedisNotFoundIdentity struct {
  errr error <- this
}

func (e *ErrRedisNotFoundIdentity) Error() string {
...
}
```

Reply:
Nop. I think we should not return `" ` as the returned value.
It is acceptable to cause nil pointer error.
But, I think each custom error struct has implemented correctly, nil pointer error will not occur.

Reply:
@hlts2 can you tell us what is the advantages of exporting the internal error?

Reply:
sorry to late reply.
At the time of the oral discussion, it was said that there was no problem with the current policy, so I will close the comment.